src/helpers/semester.ts

```diff
@@ -0,0 +1,94 @@
+const isSpringSemester = (date: Date) => {
+  const month = date.getMonth();
+  return month > 1 && month < 8;
+}
+
+export class SemesterInfo {
+  readonly springSemester: boolean;
```

Comment:
Could we call this variable `isSprintSemester` or something similar? Just seeing the name, it is not clear that this is a boolean/what it is trying to tell me.